### PR TITLE
Pokemon basic and fight card components

### PR DESCRIPTION
**Pull request by IlanSirisky**
Implemented both Basic and Fight Pokemon cards components.
including styles file that is relevant to both cards.
Added more values to the global css values.

### CODE REVIEW

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -1,21 +1,31 @@
...
+import { StyledButton } from "./styles";
 
 interface ButtonProps {
-    onClick?: () => void;
-    children: React.ReactNode;
-    type: buttonType;
-    size: buttonSize;
-    disabled?: boolean;
+  onClick?: () => void;
+  children: React.ReactNode | string;
```
Comment (ZivCohenMoveo) [inline src/components/Button/Button.tsx]: `React.ReactNode` is includes `string` type too

File: src/components/PokemonCards/PokemonFightCard.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      <HeadingLargeBold>{playerName}</HeadingLargeBold>
+      <ImagePowerContainer>
+        <StyledPokemonImage src={image} alt={cardTitle} />
+        <PowerContainer>
+          {cornerText && <XSmallRegular>{cornerText}</XSmallRegular>}
+          {topCornerIcon && <img src={topCornerIcon} alt={cardTitle} />}
+        </PowerContainer>
+      </ImagePowerContainer>
+      {subheadText && <StyledIdText>{subheadText}</StyledIdText>}
+      <HeadingLargeRegular>{cardTitle}</HeadingLargeRegular>
```
Comment (ZivCohenMoveo) [inline src/components/PokemonCards/PokemonFightCard.tsx]: Take it out to a separate component and use it here and in PokemonBasicCard.tsx component

Comment (IlanSirisky) [reply src/components/PokemonCards/PokemonFightCard.tsx]: exported to separate component

File: src/components/Button/Button.tsx (hunk 1 above)
Comment (IlanSirisky) [reply src/components/Button/Button.tsx]: updated type

**Comment by YuvalLeviMoveoHLS** [comment]
very good 👍

**Pull request merged**